packages/public/server/src/module/Common/src/Common/Filter/Slugify.php

```diff
@@ -32,8 +32,11 @@ class Slugify implements FilterInterface
      */
     protected static function slugify($text)
     {
-        $text = trim($text, " ");
         $text = preg_replace("~ +~u", '-', $text);
+        // replace non letter or digits
+        $text = preg_replace('~[^\\pL\\pM\d-]+~u', '', $text);
```

Comment:
We had the `\p{L}` filter before and removed it in #11, because it filtered too much especially in hindi and tamil names. This is now handled differently with `\p{M}`: `\p{L}` matches a character point and `\p{M}` matches accents combined to a letter, e.g. `à` is build from `\p{L}`matching `a` and `\p{M}` matching the accent (see https://www.regular-expressions.info/unicode.html).